src/Search.js

```diff
@@ -30,15 +23,16 @@ class Search extends Component {
       this.props.onEnterKey()
     }
   }
+  // Great handling of input!
 
   render() {
     return (
       <div className="searchContainer">
-      <section id={"search" + (this.props.showSearch ? '.active' : '')}>
+      <section id="search" className={this.props.showSearch ? 'active' : ''}>
```

Comment:
You can't have `.`'s in id's or classnames. the `.` is a classname selector which means you probably wanted to have the `active` in the classname not the id.

As a general rule you should always try to use classnames instead of id's when writing styles.

Reply:
Ahhh, that's where I was going wrong. I spent so much time trying to figure out the searchbox CSS issue!